core/scripts/runVoting.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Must run from the core/ directory.
+
+while sleep 60; do $(npm bin)/truffle exec ./scripts/Voting.js "$@" &> out.log; done
```

Comment:
Just curious, do you think `out.log` is more visible than CLI output in GCS or other docker contexts?

Reply:
Not sure, happy to not do the redirect if you think that's better.

Reply:
I think this is fine - just curious about your thought process.